bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVKeyboardInput.java

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+/*
```

Comment:
I think we have a similar situation here like in logreader or smartmeter.Maybe we better keep the original package name and place these files under `src/3rdparty`. Also mention they are Apache licensed in NOTICE.

Reply:
added the note in NOTICE

the files themselves are heavily refactored. would like to keep them if possible where they are right now. but will follow your guidance if they have to move